### ISSUE TITLE
building iometa on ubuntu fails: "cc: error: unrecognized command line option '-framework'"

### ISSUE THREAD

**Issue by b17fr13nds** [Siguza/iometa#4]
hello, I tried to build iometa in an ubuntu 20.04 docker container. However when I type `make`, I get the following error:
```
cc -o iometa -Wall -O3 -flto -DVERSION=1.6.6 -DTIMESTAMP="`date +'%d. %B %Y %H:%M:%S'`" -framework CoreFoundation -framework IOKit -lc++abi -Isrc  src/*.c gen/cxxsym.c
cc: error: CoreFoundation: No such file or directory
cc: error: IOKit: No such file or directory
cc: error: unrecognized command line option '-framework'
cc: error: unrecognized command line option '-framework'
make: *** [Makefile:12: iometa] Error 1
```
So I tried to remove the `-framework` flags. Now I get a very long error message: https://pastebin.com/L0pxrtZn

Can someone please describe how to build iometa on linux correctly?

Thanks, BitFriends

**Comment by Siguza** [comment]
Yeah Linux is not supported, too many dependencies on macOS internals.  
You'd have to fix up libc++abi, and find a suitable replacement for IOKit plist parsing. Maybe you could patch and compile the parsing code from IOKitUser even, but it'll be a significant amount of work.

**Comment by Siguza** [comment]
I know it's been almost 4 years, but Linux is fully supported now. You'll have to build with `clang` and you need `libc++abi` from LLVM, but... yeah, see the README for build instructions.
